lib/runtime/addon.js

```diff
@@ -341,4 +340,34 @@ export default class Addon {
     return configModule && (configModule.default || configModule);
   }
 
+  get blueprintNames() {
+    let names = fs.readdirSync(path.join(this.dir, 'blueprints'));
+    return names;
```

Comment:
It's also used by `hasBlueprint` to check if a blueprint exists without requiring and instantiating it.